src/utils/isENSNFTRecord.ts

```diff
@@ -0,0 +1,3 @@
+export default function isENSNFTRecord(avatar: string) {
```

Comment:
Would be cool to have a utils/ens file with everything ENS related (contenthash, avatars, etc) instead of split into so many files. We have like 5 files with 1 function that's only used by ENS.

Might make easier to import in the future if we need to move it to a package or share code between platforms